Sources/PointFree/Episode/Show.swift

```diff
@@ -22,16 +22,16 @@ let episodeResponse =
     <| writeStatus(.ok)
     >=> userEpisodePermission
     >=> map(lower)
-    >>> respond(
-      view: episodeView,
+    >>> _respond(
+      view: Views.episodeView(permission:user:subscriberState:episode:previousEpisodes:date:),
       layoutData: { permission, episode, currentUser, subscriberState, currentRoute in
         let navStyle: NavStyle = currentUser == nil ? .mountains(.main) : .minimal(.light)
 
         return SimplePageLayoutData(
           currentRoute: currentRoute,
           currentSubscriberState: subscriberState,
           currentUser: currentUser,
-          data: (permission, currentUser, subscriberState, episode),
+          data: (permission, currentUser, subscriberState, episode, episode.previousEpisodes, Current.date),
```

Comment:
passing in more dependencies to the view now that it lives in the `Views` module.